.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches: [ "master" ]
+    paths: [ 'scss/**' ]
+  pull_request:
+    branches: [ "master" ]
+    paths: [ 'scss/**' ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 12.x
+        cache: 'npm'
+    - run: npm ci
```

Comment:
I just realized that you have both a `package-lock.json` and a `yarn.lock`. Which package manager is preferred?

Reply:
Thanks I've removed yarn.lock in another PR!